RealmSwift/Migration.swift

```diff
@@ -127,7 +127,7 @@ public final class Migration {
     - returns: The created object.
     */
     @discardableResult
-    public func createObject(ofType typeName: String, populatedWith value: Any = [:]) -> MigrationObject {
+    public func create(_ typeName: String, value: Any = [:]) -> MigrationObject {
```

Comment:
It's kind of unfortunate this doesn't read as a sentence anymore. `create(:with:)` or `create(:from:)` might be okay, though not great.